test/integration/container/multimaster_test.go

```diff
@@ -211,6 +211,16 @@ data:
 `
 
 func TestMultimasterSetup(t *testing.T) {
+	node_os := os.Getenv("NODE_OS")
+
+	if node_os == "" {
+		node_os = "centos"
+	}
+	node_version := "18.04"
+	if node_os == "centos" {
+		node_version = "7"
+	}
+
```

Comment:
Nit: this can be simplified a bit:
```
node_os, node_version := os.Getenv("NODE_OS"), "18.04"
if node_os != "ubuntu" {
   node_os, node_version = "centos", "7"
}
```

Reply:
That does indeed simplify it more!